src/less/items.css

```diff
@@ -0,0 +1,660 @@
+:root {
```

Comment:
I don't think this is meant to be here

Reply:
Yeah you're correct, dunno how this ended up there. I've removed it